scripts/release

```diff
@@ -2,7 +2,8 @@
 # This script updates lepidopter version and ChangeLog files
 set -e
 
-VERSION_FILE="lepidopter-fh/etc/lepidopter_version"
+source conf/lepidopter-image.conf
+UPDATE_INCREMENT=$(awk '{print $1+1}' ${UPDATE_VERSION_FILE})
```

Comment:
This is not always going to be (as is the case of this release 3+1 != 8). It's probably more robust to fetch the latest version number from the repo or rely on manual updating.